src/jquery.mustache.js

```diff
@@ -127,17 +127,23 @@
 	 * Loads the external Mustache templates located at the supplied URL and registers them for later use.  This method
 	 * returns a jQuery Promise and also support an `onComplete` callback.
 	 *
-	 * @param url			URL of the external Mustache template file to load.
+	 * @param url|settings  URL of the external Mustache template file to load, or a set of key/value pairs that 
+	 *						configure the Ajax request. All settings are optional except url. DataType setting will be
+	 *						overwrite.
 	 * @param onComplete	Optional callback function which will be invoked when the templates from the supplied URL
 	 *						have been loaded and are ready for use.
 	 * @returns				jQuery deferred promise which will complete when the templates have been loaded and are
 	 *						ready for use.
 	 */
 	function load(url, onComplete) {
-		return $.ajax({
-				url: url,
-				dataType: options.externalTemplateDataType
-			}).done(function (templates) {
+
+		var obj = {};
+
+		obj = typeof url === 'object' ? obj = url: obj.url = url;
+
+		obj.dataType = options.externalTemplateDataType;
```

Comment:
Like this:
```
obj.dataType = obj.dataType || options.externalTemplateDataType;
```

If user was settup a dataType, it will be used and if dont default DataType will be used.